### DISCUSSION TITLE
Welcome to Bitmaps2Video Discussions!

### DISCUSSION THREAD

**Discussion by rmesch** [rmesch/Bitmaps2Video#5] [General]
👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by yarockiisergei** [comment]
Hello. I looked at the Bitmaps2Video example. I want to ask you for help, my task is to record a video to a file. I implemented getting bitmap images from an Android camera. But I don't understand how to organize recording to a file in mp4 format. I really hope for your help.
